Objects/bytes_methods.c

```diff
@@ -556,11 +556,55 @@ find_internal(const char *str, Py_ssize_t len,
     return res;
 }
 
+#define FIND_CHUNK_SIZE 10000
```

Comment:
Is it fine to define this twice, or should I put it in a different file?

Reply:
It might be better leaving this within a function. What if someone (after some changes) decides to have different sizes for `find` and `rfind` in the future? As I said, very unlikely, but decision whether to have a global constant can be left until there is more information.

Reply:
Then you set `RFIND_CHUNK_SIZE` to a specific value.